miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java

```diff
@@ -242,4 +249,43 @@ public PoolDto updatePool(@PathVariable Long poolId, @RequestBody PoolDto pool)
     return "[" + LimsUtils.join(names, ",") + "]";
   }
 
+  @RequestMapping(value = "/picker/search")
+  @ResponseBody
+  public PoolPickerResponse getPickersBySearch(@RequestParam("platform") String platform, @RequestParam("query") String query)
+      throws IOException {
+    PlatformType platformType = PlatformType.get(platform);
+    return getPoolPickerWithFilters(100,
+        PaginationFilter.platformType(platformType),
+        PaginationFilter.query(query));
+  }
+
+  @RequestMapping(value = "/picker/readytorun")
+  @ResponseBody
+  public PoolPickerResponse getPickersByReadyToRun(@RequestParam("platform") String platform,
+      @RequestParam("readyToRun") boolean readyToRun) throws IOException {
+    PlatformType platformType = PlatformType.get(platform);
+    return getPoolPickerWithFilters(100, PaginationFilter.platformType(platformType),
+        PaginationFilter.readyToRun(readyToRun));
+  }
+
+  @RequestMapping(value = "/picker/recent")
+  @ResponseBody
+  public PoolPickerResponse getPickersBySearch(@RequestParam("platform") String platform)
+      throws IOException {
+    PlatformType platformType = PlatformType.get(platform);
+    return getPoolPickerWithFilters(20,
+        PaginationFilter.platformType(platformType));
+  }
+
+  private PoolPickerResponse getPoolPickerWithFilters(Integer limit, PaginationFilter... filters) throws IOException {
+    PoolPickerResponse ppr = new PoolPickerResponse();
+    ppr.populate(poolService, false, "lastModified", limit, this::poolTransform, filters);
+    return ppr;
+  }
+
+  private PoolPickerEntry poolTransform(Pool pool) throws IOException {
+    List<PoolOrderCompletionDto> completions = poolOrderCompletionService.getByPoolId(pool.getId()).stream()
+        .map(completion -> Dtos.asDto(completion)).collect(Collectors.toList());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

Reply:
Changing `completion -> Dtos.asDto(completion)` to `Dtos::asDto` will satisfy Sonar